libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -70,6 +70,22 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("VOLT2_MIN",     5,     AP_Arming,  _min_voltage[1],  0),
 
+    // @Param: OBAUTHINT
+    // @DisplayName: Offboard Authorization Interval
```

Comment:
if the param is OB* then lets' have the displayname and desc emphasize the other B like "Off-Board.."